src/cmd/initContainer.go

```diff
@@ -405,7 +393,6 @@ func configureUsers(targetUserUid int,
 			"--append",
 			"--groups", sudoGroup,
 			"--home", targetUserHome,
-			"--shell", targetUserShell,
```

Comment:
We still need to set up the user's default shell inside the container to be something sane as the initial starting point or *factory state*.  Otherwise, for the vast majority of users, who don't change their default shell and just use Bash, the default shell inside the container will become `/bin/sh`.  That will be disturbing.

The most sane default inside the container is to match the host, because very few people will want separate shells across their containers.

We need to figure out a way to persistently record (across reboots) that the container was already set up as per this factory state, and that everything from now on is a local change made by the user. This way, `toolbox init-container` can avoid overwriting the user's local changes.

Removing this persistent record (whatever it is) should reset the container back to its factory state. This can be very helpful to debug problems and/or recover broken containers by going back to a known-good state.

Reply:
I restored the `--shell` option from `toolbox init-container` as you requested.